src/encoded/commands/deploy.py

```diff
@@ -21,6 +21,19 @@
 REPO_DIR = f"{str(Path().parent.absolute())}"
 
 
+def _fetch_ami_id(bucket: str = "packer-ami-id-and-log", key: str = "ami-id/current_ami_id.txt") -> str:
+    """Fetch the current ami-id from S3.
+    Args:
+        bucket: name of S3 bucket where ami-id is stored
+        key: location of the ami-id file within the bucket
+    Returns:
+        ami_id: string containing the contents current ami id S3 object.
+    """
+    s3 = boto3.resource("s3")
+    ami_id = s3.ObjectSummary(bucket_name=bucket, key=key).get()["Body"].read().decode().strip()
+    print(f"Fetched ami: {ami_id}")
```

Comment:
Might wanna add the image id to the instance tags in the 'def _tag_ec2_instance' function